frontend/src/main.js

```diff
@@ -6,10 +6,16 @@ import BootstrapVue from 'bootstrap-vue'
 import axios from 'axios'
 import VueAxios from 'vue-axios'
 
+import EmptyLayout from './layouts/EmptyLayout'
+import DefaultLayout from './layouts/DefaultLayout'
+
 Vue.use(BootstrapVue)
 axios.defaults.withCredentials = true
 Vue.use(VueAxios, axios)
 
+Vue.component('empty-layout', EmptyLayout)
```

Comment:
Sind diese Zeilen hier, damit die Layouts in jedem Fall eager loaded werden? Oder funktioniert es nicht ohne?

Reply:
Umgebaut - Lazy laden